katas/content/deutsch_jozsa/index.md

```diff
@@ -17,9 +17,11 @@ This kata introduces you to Deutsch-Jozsa algorithm - one of the most famous alg
 
 **What you should know to start working on this kata:**
 
-- Basic knowledge of single-qubit gates
-- Basic knowledge of quantum measurements
-- Deutsch algorithm - the single-qubit variant of Deutsch-Jozsa algorithm. If you're not familiar with it, you can find it in the Deutsch algorithm kata.
+- Basic single-qubit gates
+- Quantum measurements
+- Deutsch algorithm - the single-qubit variant of Deutsch-Jozsa algorithm. 
+
+If you need a refresher on these topics, you can check out the previous katas.
```

Comment:
is it worth mentioning _which_ katas? same for the other time this sentence appears

Reply:
We discussed this previously (not sure if this PR or an earlier one). The list of all relevant previous katas is going to grow unmanageable once we move out from the first few katas (we have over 20 katas now, and the first ten or so introduce the basics that are used over and over again), so we decided against including a specific list.